.travis.yml

```diff
@@ -12,10 +12,5 @@ before_script:
   - sudo apt-get -qq -y install podman
 
 script:
-- make
-- make vendorcheck
-- make cross-lint
-- make cross
-- make test
-- make build_integration
+- make check
```

Comment:
shouldn't this added to other CI as well like circleCI, appveyor and centos one?